models/enterprise_sonic/ssh/sonic_ssh.yml

```diff
@@ -0,0 +1,67 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'community',
+        'license': 'Apache 2.0'
+    }
+NETWORK_OS: sonic
+RESOURCE: ssh
+COPYRIGHT: Copyright 2024 Dell Inc. or its subsidiaries. All Rights Reserved
+DOCUMENTATION: |
+    module: sonic_ssh
+    version_added: '2.1.0'
+    short_description: Manage SSH configurations on SONiC
+    description:
+      - This module provides SSH configuration management to specify the 
+        algorithms used for SSH connection in devices running SONiC.
+    author: 'Balasubramaniam Koundappa(@balasubramaniam-k)'
+    options:
+      config:
+        description: SSH clients and servers use the following configurations for connecting with the server.
+        type: dict
+        suboptions:
+          client:
+            description:
+              SSH Client configuration
+            type: dict
+            suboptions:
```

Comment:
Please provide descriptions for each of these suboptions that state they are specified as comma separated lists. Specify the allowed values in the comma separated list for each of the options. It would be good to do this in a manner similar to the suboptions descriptions in [Model changes for REST server cipher suite configuration.](https://github.com/ansible-network/resource_module_models/pull/276).

Reply:
Addressed.